crates/rethnet_rpc_server/tests/integration_tests.rs

```diff
@@ -283,3 +291,17 @@ async fn test_set_storage_at_success() {
     )
     .await;
 }
+
+#[tokio::test]
+async fn test_sign() {
+    // the expected response for this test case was created by submitting the same request to a
+    // default-configured instance of Hardhat Network.
+    verify_response(
+        &start_server().await,
+        MethodInvocation::Eth(EthMethodInvocation::Sign(
+            Address::from_str("0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266").unwrap(),
+            bytes::Bytes::from(hex::decode("deadbeef").unwrap()).into(),
```

Comment:
Add a use statement for `rethnet_eth::Bytes`

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4149/commits/73021c054ec3681f3f932561cb1d29155b4b5c0a